src/vs/editor/contrib/find/findWidget.ts

```diff
@@ -320,7 +320,6 @@ export class FindWidget extends Widget implements IOverlayWidget, IHorizontalSas
 		if (e.searchString || e.matchesCount || e.matchesPosition) {
 			let showRedOutline = (this._state.searchString.length > 0 && this._state.matchesCount === 0);
 			dom.toggleClass(this._domNode, 'no-results', showRedOutline);
```

Comment:
You removed an empty line, and added an empty line. I do not realy see a value in that.